### DISCUSSION TITLE
Intelichlor IC40 RS485

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#539] [RS485] [answered]
@rstrouse @tagyoureit  You guys are really amazing.  I'm sure everyone here can agree what a great job you do to help everyone.  You'll have to let me know how to send you each a bottle of bourbon or some HW to play with.

I moved my controller from a version of njsPC that was two years old, I can get the version but I doubt it has relevance, to the latest njsPC and REM etc, as you're aware with the onslaught of threads I've opened over the last two days.  That being said, I had recently installed an IC40 and was communicating via RS485 like it should prior to the migration.

After moving over to the new hardware and platform yesterday, the IC40 is not seen by njsPC.  I've moved the IC40 over to its own RS485 USB adapter so see if I could `od -x < /dev/ttyUSB0` and I'm getting nothing.  I've even gone so far as to run a new RS485 wire to the IC40's transformer.

Is there a reset I can do on the IC40?

**Comment by tagyoureit** [comment]
The IC40 won't be chatty unless asked for its status.  Have you added the IC40 under the chemistry section with Nixie as the controller?

**Comment by rstrouse** [comment, marked as answer]
Older versions of njsPC used to send out probes to see if the device exists.  However, this can be problematic when there is no device on the other end.  Did you go to the chemistry tab and add your chlorinator?  njsPC supports multiple chlorinators over multiple RS485 buses so you have to tell it about its existence.

In dashPanel click on the config gears in the upper right.  Then choose the Chemistry tab.  Click Add Controller and on the Chlorinator dropdown choose which controller should be in control of the Chlorinator.  Yes unlike all of the commercial controllers on the market Nixie can make use of multiple chlorinators, even from different vendors (if they have RS485 control).
![image](https://user-images.githubusercontent.com/47839015/179416172-a2a6cd80-d104-4295-9cd2-cc643383d8ea.png)


@tagyoureit beat me to it.

**Reply by orrious** [reply]
I had already done this.  I went back in deleted it, re-added it and it works.  :smh:

Thanks again!
